ornate_frontend/src/context/AuthContext.js

```diff
@@ -18,6 +18,16 @@ export const AuthContextProvider = ({ children }) => {
     user: null,
   });
 
+  // Checking weather the user is there in the local storage and login the user again.
+
+  useEffect(() => {
+    const user = JSON.parse(localStorage.getItem("user"));
```

Comment:
Every JavaScript file that is loaded on your domain has access to Local Storage. If malicious JavaScript code is added by you or your dependencies, they can retrieve user data or tokens you use to authenticate with APIs.

Reply:
Made the required commits.